src/server/console/logind_console_services.cpp

```diff
@@ -844,6 +845,7 @@ void mir::LogindConsoleServices::request_lock(
     GObject*,
     gpointer ctx) noexcept
 {
+    mir::log_debug("Received Lock() signal for this session from logind");
```

Comment:
This probably wants to be `log_info`; this is the interaction of Mir with another system component, something that a user is plausibly going to know about and be interested in.
```suggestion
    mir::log_info("Received Lock() signal for this session from logind");
```